cmd/gctcli/commands.go

```diff
@@ -3521,31 +3521,31 @@ func getExchangeTickerStream(c *cli.Context) error {
 	}
 }
 
-var getAuditEventCommand = cli.Command{
+var getAuditEventCommand = &cli.Command{
 	Name:      "getauditevent",
 	Usage:     "gets audit events matching query parameters",
 	ArgsUsage: "<starttime> <endtime> <orderby> <limit>",
 	Action:    getAuditEvent,
 	Flags: []cli.Flag{
-		cli.StringFlag{
+		&cli.StringFlag{
 			Name:        "start, s",
```

Comment:
These will need to be adjusted as per https://github.com/urfave/cli/blob/master/docs/migrate-v1-to-v2.md#flag-aliases-are-done-differently